x-pack/plugins/task_manager/server/task_type_dictionary.ts

```diff
@@ -8,6 +8,17 @@
 import { TaskDefinition, taskDefinitionSchema, TaskRunCreatorFunction } from './task';
 import { Logger } from '../../../../src/core/server';
 
+/**
+ * Types that are no longer registered and will be marked as unregistered
+ */
+export const REMOVED_TYPES: string[] = [
+  // for testing
+  'sampleTaskRemovedType',
+
+  // deprecated in https://github.com/elastic/kibana/pull/121442
+  'alerting:siem.signals',
```

Comment:
@madirey @marshallmain 

Tagging you on this PR for awareness. We adding an explicit registry of removed task types that will be marked as `unrecognized`. Adding `alerting:siem.signals` due to [this PR](https://github.com/elastic/kibana/pull/121442) that removed that rule type. There should be no change to the outcome of that PR (meaning old `siem.signals` rule types will still be marked as `unrecognized`)